### PR TITLE
Support/commenting

### LINKED ISSUE #93: Commenting in Javascript and Java
Java and Javascript should have comments. Overridden methods should have comments too.

### PR DESCRIPTION
**Pull request by rubenwiersma**
I've added comments to methods in Java and JavaScript that still needed comments. I have not commented controllers or state configurations, as their functionality is clear from a structural perspective and adding comments would be redundant.
Fixes #93

### CODE REVIEW

File: src/main/webapp/app/account/reset/request/reset.request.state.js [hunk 1]
```diff
@@ -7,6 +7,10 @@
 
     stateConfig.$inject = ['$stateProvider'];
 
+    /**
+     * Configuration for the Request Reset state.
+     * @param $stateProvider
```
Comment (ChrisBerg) [inline src/main/webapp/app/account/reset/request/reset.request.state.js]: No comments by parameters?

File: src/main/webapp/app/account/reset/request/reset.request.controller.js [hunk 2]
```diff
@@ -7,6 +7,13 @@
 
     RequestResetController.$inject = ['$timeout', 'Auth'];
 
+    /**
+     * Controller for the Request Reset state.
+     * A user can reset his password here.
+     * @param $timeout
+     * @param Auth
+     * @constructor
```
Comment (ChrisBerg) [inline src/main/webapp/app/account/reset/request/reset.request.controller.js]: No comments by parameters?

**Comment by hungnguyen94** [comment]
Good job! The fabric-map directive is still missing documentation though. Should that be added here too?

**Comment by rubenwiersma** [comment]
@hungnguyen94 The fabric-map directive has documentation, look closely 😉

**Comment by hungnguyen94** [comment]
@rubenwiersma Oops, I was looking in the wrong place.

File: src/main/java/nl/tudelft/thefirstorder/service/impl/CueServiceImpl.java [hunk 3]
```diff
@@ -54,6 +54,11 @@ public Cue save(Cue cue) {
         return cueRepository.findAll(pageable);
     }
 
+    /**
+     * Find a cues by their script id.
```
Comment (trombomart) [inline src/main/java/nl/tudelft/thefirstorder/service/impl/CueServiceImpl.java]: *Find cues (without a)

File: src/main/webapp/app/scripting/scripting.controller.js [hunk 4]
```diff
@@ -28,6 +28,9 @@
             }
         });
 
+        /**
+         * Update cues by cuerying Cue.
```
Comment (trombomart) [inline src/main/webapp/app/scripting/scripting.controller.js]: *querying?

**Comment by trombomart** [comment]
Nice job. A few little spelling mistakes. After that I will merge it.

**Pull request merged**